packages/DecorativeIcon/svgs/CartTeam.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import SVGIcon from '../SVGIcon'
+
+const CartTeam = props => (
```

Comment:
Please clean this icon up. It should look similar to the other decorative icon svgs.

Ex. https://github.com/telus/tds-core/blob/master/packages/DecorativeIcon/svgs/Attention.jsx

It's only a single path, we should be able to remove most of the other tags.
The SVGIcon component controls the icon colour, so please remove any hard coded colour values in the SVG.